src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -511,6 +471,22 @@ export default {
 	beforeDestroy() {
 		this.destroyViewportObserver();
 	},
+	created() {
+		try {
```

Comment:
This will throw an invariant error when the fragment is in the cache but missing fields. Lets just swallow the error and do nothing. KivaClassicBasicLoanCard just does what it used to do before this PR in that case

Reply:
Since the fragment now has the extra fields, in what case will the loan in the cache have missing fields?

Reply:
I found at least one case, on the homepage (i guess now old homepage) the multi category carousel uses a query which just gets the loan ID. `src/components/LoanCollections/KivaClassicMultiCategoryCarousel.vue:121`

Reply:
That's correct b/c we don't want all the loan data prefetched.